draft-marx-qlog-event-definitions-quic-h3.md

```diff
@@ -73,6 +73,8 @@ informative:
 
 --- abstract
 
+ === beginning the split into two separate documents for quicwg adoption ===
```

Comment:
Is the intent to delete this file?

Reply:
Indeed, this one is removed and replaced by the two new ones.

Reply:
so uhh, why isn't it `git rm`'d ? :D